fetch/data-urls/resources/data-urls.json

```diff
@@ -0,0 +1,199 @@
+[
+  ["data://test/,X",
+   "text/plain;charset=US-ASCII",
+   [88]],
+  ["data://test:test/,X",
+   null],
```

Comment:
Why does this test case fail? Is it the URL parser that returns an error?

Reply:
This URL loads an `X` document in Firefox and Chromium. (It doesn’t in Edge, but Edge appears to reject any `data:` URL with a MIME type parsing error.)

Reply:
Yeah.